jsign-core/src/test/resources/keystores/generate-private-key.sh

```diff
@@ -8,6 +8,7 @@ openssl genrsa -aes128 -traditional -passout pass:password -out privatekey-encry
 openssl pkcs8 -topk8       -in privatekey-encrypted.pkcs1.pem -out privatekey-encrypted.pkcs8.pem -passin pass:password -passout pass:password -v2 aes128
 openssl pkcs8 -topk8       -in privatekey-encrypted.pkcs1.pem -out privatekey.pkcs8.pem -passin pass:password -nocrypt
 openssl pkcs8 -traditional -in privatekey.pkcs8.pem           -out privatekey.pkcs1.pem -nocrypt
+cp privatekey.pkcs1.pem privatekey.pkcs1.pem.key
```

Comment:
No need to copy the file here, this can be done in the test method

Reply:
Good point, I think that's indeed cleaner. I'm pushing a commit to do that. Thanks!